src/InvoiceNinja/Models/Statics.php

```diff
@@ -6,16 +6,16 @@ class Statics extends AbstractModel
 {
     public static $route = 'static';
 
-    public static function all()
+    public static function all($raw = false)
```

Comment:
I think this needs to be changed to true otherwise it would change the behavior

Reply:
But the expected behaviour is that a correct JSON response is returned when `all()` is called directly. Like any other models.

Reply:
Ok, works for me